### ISSUE TITLE
The movement-controls become inverted when adjusting the camera rig's Y-axis rotation

### ISSUE THREAD

**Issue by frankangelone** [c-frame/aframe-extras#271]
I am setting the following `rotation` settings for my camera rig: 

```
cameraRigEl = document.createElement('a-entity');
cameraRigEl.setAttribute("id", "rig");
cameraRigEl.setAttribute("movement-controls", "");
cameraRigEl.setAttribute("rotation", "0, 180, 0");
```
When I try to use the WASD keys within my A-FRAME scene, the W key behaves like the S key (and vice-versa with the same applying to the A and D keys). 

I have the `rotation` of the Y-axis for the camera rig set to 180 so that I'm facing my 3D model on page load of the A-FRAME scene.

**Comment by donmccurdy** [comment]
This should have been fixed in 4.2.0, which version are you on?

**Comment by donmccurdy** [comment]
Hm good question.. if you're installing via npm you could use [npm-check-updates](https://www.npmjs.com/package/npm-check-updates). Or there's an [RSS feed](https://github.com/donmccurdy/aframe-extras/releases.atom) and (for more significant releases) a [changelog](https://github.com/donmccurdy/aframe-extras/blob/master/CHANGELOG.md).
